### ISSUE TITLE
Elevated CD4-positive, CD25-positive regulatory T cell count HP:0030337

### ISSUE THREAD

**Issue by adamklocperk** [obophenotype/human-phenotype-ontology#11259]
**HPO term**
Elevated CD4-positive, CD25-positive regulatory T cell count HP:0030337

**Suggested revision of textual definition**
Please rename as "Increased proportion of Treg cells".

Description: Abnormal increase of the regulatory (Treg) CD4+ T cell subpopulation, commonly characterized by the CD127lowCD25hi phenotype, with the optional additional positivity for FoxP3, measured as percentage of total CD4+ T cells in the blood, compared to a reference range for a given sex and age-group.

**Comment by pnrobinson** [comment]
@adamklocperk HP:0030337 was merged into the following term

[Increased proportion of regulatory T cells HP:0020112](https://hpo.jax.org/browse/term/HP:0020112)

I have added Increased proportion of Treg cells as a synonym to the latter term (let's not use it as a primary label, because "Treg" is a kind of slang term that might not be obvious to all users of HPO (e.g., non-immunologists).

Please reopen this issue if the above is incorrect.
